JS/Lesson30/gulcin/assignment-files/job-board-app/src/components/Card-component/Card.jsx

```diff
@@ -0,0 +1,31 @@
+import { jobData } from '../../data';
+import './card.css';
+
+
+
+export const Cards = () => {
+    return (
+        <div className='cards-holder'>
+            {jobData.map((job) => (
+                <div className='job-card'>
+                    <div className='img-and-info'>
+                        <img className='card-img' src={job.logo} alt='Company Logo' />
+                        <div className='job-info'>
+                            <div className='company-area'><span>{job.company}</span>{job.new && <span className='tag-new'>NEW!</span>}{job.featured && <span className='tag-featured'>FEATURED</span>}</div>
+                            <div className='job-definition'>{job.position}</div>
+                            <div className='info-bottom'><span>{job.postedAt}</span><span>&#183;</span><span>{job.contract}</span><span>&#183;</span><span>{job.location}</span></div>
+                        </div>
+                    </div>
+                    <ul className='job-tags'>
+                        <li className='job-tag' key={job.role}>{job.role}</li>
```

Comment:
I'd recommend to make a component for the job tag, then you can reuse it when needed. In React, we usually split the code into small reusable parts (components) and combine them later as needed.